### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by emrealtinoz** [mx0c/Corsair-Void-Pro-Battery-Overlay#16]
Hi, i use rgb elite wireless and i changed my pid but when i run the app it says Error.

![resim_2022-02-26_112538](https://user-images.githubusercontent.com/78230538/155836137-4776acaf-dcdc-4ba8-96fe-71746433d08d.png)

**Comment by mx0c** [comment]
What did you change the pid into?

**Comment by emrealtinoz** [comment]
> What did you change the pid into?

0A51

**Comment by mx0c** [comment]
hmm should work, have you tried the predefined option?

**Comment by emrealtinoz** [comment]
> ave you tried the predefined option?

Yes i did. I chose my headphone. I use icue 3, maybe this is the problem. Is app compatible with icue 3?

**Comment by mx0c** [comment]
The app doesn't communicate with ICUE and therefore the version is irrelevant. Can't explain why it doesn't work, sorry :( Maybe you could try to debug it yourself. Let me know if you need help with it.

**Comment by anasbinwaquar** [comment]
I am having the same issue. Please tell if there's any fix for it? I have HS70 Pro It does the same thing. Says error and doesn't show battery % :(
![image](https://user-images.githubusercontent.com/60139674/196989007-d26fe434-bbf8-4f7e-ba18-b7c8d3fe2dbd.png)

**Comment by mx0c** [comment]
@anasbinwaquar have you tried following the troubleshooting section of the readme ?

**Comment by anasbinwaquar** [comment]
Yes but i dont see any hardware ID it just says string. @mx0c 
![image](https://user-images.githubusercontent.com/60139674/197239058-cf8482a9-8714-4bf3-b14d-9bbdb8fc0a8d.png)
What should i do now? It used to run fine before i fresh installed windows 11.

Edit i checked PID and its set correctly but still does not recognize my device. 
![image](https://user-images.githubusercontent.com/60139674/197242162-9e78f84e-6eb1-4237-96f9-14bba875084d.png)
What should i do in this case?
